editions/tw5.com/tiddlers/widgets/EventCatcherWidget.tid

```diff
@@ -48,6 +48,8 @@ The following variables are made available to the actions:
 |`event-type` |The type property of the JavaScript event |
 |`event-detail-*` |Any properties in the detail attribute of the event are made available with the prefix `event-detail-` |
 |`tv-popup-coords` |A co-ordinate string that can be used with the ActionPopupWidget to trigger a popup at the DOM node matching the selector where the event originated |
+|`tv-widgetnode-width` |<<.from-version "5.2.3">> `offsetWidth` of the DOM node created by the eventcatcher widget |
+|`tv-widgetnode-height` |<<.from-version "5.2.3">> 
```

Comment:
formatting issues plus missing docs for `tv-widgetnode-height`.

Reply:
Now this should be resolved. Thanks @saqimtiaz